chronograf.go

```diff
@@ -702,6 +704,56 @@ type LayoutsStore interface {
 	Update(context.Context, Layout) error
 }
 
+// ProtoboardMeta is the metadata of a Protoboard
+type ProtoboardMeta struct {
```

Comment:
TBH, not thrilled with the `Proto*` naming.

Reply:
other ideas?

Reply:
I've grown to like the name